### PR TITLE
Umbraco Turkiye Hacktoberfest meetup - organizing the event and speaking at the event

### PR DESCRIPTION
**Pull request by NurhakKaya**
Organizing Umbraco Turkiye Hacktoberfest meetup with Busra and speaking at the event with a live Hacktoberfest contribution demo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ Contributions:
 - 2023-10-14 - [Lotte Pitcher](https://github.com/LottePitcher) - [Hacktoberfest event](https://www.meetup.com/umbraco-virtual/events/296099686/) - Organiser of all day virtual Umbraco hacktoberfest hackathon
 - 2023-10-14 - [Carole Logan](https://github.com/crgrieve) - Blog post [Using Umbraco Forms Headless API](https://carole.dev/blog/using-umbraco-forms-headless-api/)
 - 2023-10-17 - [Busra Sengul](https://github.com/busrasengul) - Organizing [Turkiye Umbraco User Group Meetup](https://www.meetup.com/turkiye-umbraco-user-group-kullanicilari-grubu/events/295947641/) Hacktoberfest edition with a talk about Content Delivery API
+- 2023-10-17 - [Nurhak Kaya](https://github.com/NurhakKaya) - Organizing [Turkiye Umbraco User Group Meetup](https://www.meetup.com/turkiye-umbraco-user-group-kullanicilari-grubu/events/295947641/) Hacktoberfest edition with a talk about Hacktoberfest: A contribution guide to Umbraco and other open-source projects
```
Comment (busrasengul) [inline README.md:51]:
```suggestion
- 2023-10-17 - [Nurhak Kaya](https://github.com/NurhakKaya) -  [Turkiye Umbraco User Group Meetup](https://www.meetup.com/turkiye-umbraco-user-group-kullanicilari-grubu/events/295947641/) Hacktoberfest edition with a talk about Hacktoberfest: A contribution guide to Umbraco and other open-source projects
```

Comment (busrasengul) [reply README.md:51]: Sorry Nurhak but I'm the only organizer TUUG meetup. Although I appreciate your help, I can only accept the talk, not the organized part of this contribution :)

**Review by busrasengul — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (busrasengul) [reply README.md:51]: Tiny change ;) But good to merge after.

Comment (NurhakKaya) [reply README.md:51]: Well I did organize it with you and helped a lot more than maybe I should have done but no problem, I can update the text to make you happy.

Comment (busrasengul) [reply README.md]: As I said, thank you for your help, but that just doesn't qualify as organizing. I don't intend to talk down any of what you did, but as an asked speaker the things you did were only for your talks. Thanks again :)

**Pull request merged**